### ISSUE TITLE
Undefined vnode.attrs in child components in 1.0

### ISSUE THREAD

**Issue by ACXgit** [StephanHoyer/mithril-node-render#47]
Hello, with the latest version I'm unable to pass down data via `vnode.attrs` to child components.
Commit https://github.com/StephanHoyer/mithril-node-render/commit/ddf38d10d613895e2c5c1dd2c15a50ccc5df3d26 seems to be the culprit - previous versions are working as expected.

Test code to reproduce the issue:

package.json:
```
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "mithril": "https://github.com/lhorie/mithril.js/tarball/48d786b08553d5463d8ed14e6fb6c95610801520",
    "mithril-node-render": "github:stephanhoyer/mithril-node-render#rewrite"
  }
}
```

index.js:
```
global.window = require('mithril/test-utils/browserMock')();
global.document = window.document;

const m = require('mithril');
const render = require('mithril-node-render');

const component = {
    view: () => {
        return m('strong', m(subComponent, {text: 'test'}));
    }
}

const subComponent = {
    oninit: (vnode) => {
        console.log('vnode.attrs: ', vnode.attrs);
    },
    view: (vnode) => {
        return m('span', vnode.attrs.text);
    }
}

const html = render(m(component));
console.log('HTML: ', html);
```

**Comment by HendrikRoth** [comment]
Same issue here

**Comment by StephanHoyer** [comment]
fixed with #48

**Comment by ACXgit** [comment]
Tested on a fairly complex isomorphic app and it works again as expected. Thank you sirs 👍
